diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
index 6d9bcaa..268927d 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
@@ -7,7 +7,11 @@
 #import <jsi/jsi.h>
 
 #import <React/RCTBridgeModule.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 using namespace facebook;
 using namespace react;
diff --git a/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h b/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
index d7a5fdb..6f5f292 100644
--- a/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
+++ b/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
@@ -4,7 +4,11 @@
 
 #ifdef __cplusplus
 
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 #include <ExpoModulesCore/MainThreadInvoker.h>
 
 #include <jsi/jsi.h>
