diff --git a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.h b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.h
index b957a3e6d5ee9c2f031a852dc1d6ed7c96f6b5f7..ac90fbf68ec32698ad4cd8d5bd88037bb0c84c10 100644
--- a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.h
+++ b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.h
@@ -8,6 +8,12 @@
 
 // TODO: Replace this with <ReactCommon/TurboModuleUtils.h> after we upgrade to RN 0.66
 #import <EXAV/CallbackWrapper.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+
 
 namespace jsi = facebook::jsi;
 using CallInvoker = facebook::react::CallInvoker;
diff --git a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/EXAV+AudioSampleCallback.mm b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/EXAV+AudioSampleCallback.mm
index 3b8a6f298d5bab2339f8c36b951001605f96ae1e..155381b0010f702c0df1a1a1a56b6ded1a9f7b06 100644
--- a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/EXAV+AudioSampleCallback.mm
+++ b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/EXAV+AudioSampleCallback.mm
@@ -9,6 +9,12 @@
 #import <EXAV/EXAVPlayerData.h>
 #import <EXAV/EXAudioSampleCallback.h>
 #import <EXAV/AudioSampleCallbackWrapper.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+
 
 namespace jsi = facebook::jsi;
 using CallInvoker = facebook::react::CallInvoker;
diff --git a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.mm b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.mm
index 1c74d5046347c989e5da326e00dfd23e81b20ba5..de8dcf56edf71f7771ce07228badeb2233da6cc9 100644
--- a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.mm
+++ b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/AudioSampleCallbackWrapper.mm
@@ -1,6 +1,12 @@
 // Copyright 2017-present 650 Industries. All rights reserved.
 
 #import <EXAV/AudioSampleCallbackWrapper.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+
 
 namespace expo {
 namespace av {
diff --git a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/Utils/CallbackWrapper.h b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/Utils/CallbackWrapper.h
index 41148b961b09ca2555395efd2052372159ccc57e..7b32268c1daf593b32666968e1bfc0c7cf3270c8 100644
--- a/node_modules/expo-av/ios/EXAV/AudioSampleCallback/Utils/CallbackWrapper.h
+++ b/node_modules/expo-av/ios/EXAV/AudioSampleCallback/Utils/CallbackWrapper.h
@@ -15,7 +15,11 @@
 #include <unordered_set>
 
 #import <jsi/jsi.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 using namespace facebook;
 using CallInvoker = facebook::react::CallInvoker;
diff --git a/node_modules/expo-av/android/src/main/cpp/JAVManager.h b/node_modules/expo-av/android/src/main/cpp/JAVManager.h
index 6107c0ab93652fbc3677f25ba6e7914edd25ced5..81c7b464d9bd2f6e4f16aaabea872389da1f2b24 100644
--- a/node_modules/expo-av/android/src/main/cpp/JAVManager.h
+++ b/node_modules/expo-av/android/src/main/cpp/JAVManager.h
@@ -9,7 +9,11 @@
 #include <memory>
 
 #include <ReactCommon/CallInvokerHolder.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 #include "JPlayerData.h"
 
diff --git a/node_modules/expo-av/android/src/main/cpp/JAVManager.cpp b/node_modules/expo-av/android/src/main/cpp/JAVManager.cpp
index def69a0b89774339d8fbb9355831764076cbbdae..736675be25d44d1f1ed74315ca66db2d2f5ef94f 100644
--- a/node_modules/expo-av/android/src/main/cpp/JAVManager.cpp
+++ b/node_modules/expo-av/android/src/main/cpp/JAVManager.cpp
@@ -11,7 +11,11 @@
 #include <string>
 
 #include <ReactCommon/CallInvokerHolder.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 #include <android/log.h>
 
