diff --git a/node_modules/expo/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h b/node_modules/expo/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
index 407f57f..86b41d0 100644
--- a/node_modules/expo/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
+++ b/node_modules/expo/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
@@ -8,6 +8,11 @@
 #import <React/RCTBridgeModule.h>
 #import <ReactCommon/TurboModuleUtils.h>
 #import <ExpoModulesCore/ObjectDeallocator.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace jsi = facebook::jsi;
 namespace react = facebook::react;
