diff --git a/node_modules/expo-modules-core/android/src/main/cpp/JSIContext.h b/node_modules/expo-modules-core/android/src/main/cpp/JSIContext.h
index 20b65d7..f29cb7d 100644
--- a/node_modules/expo-modules-core/android/src/main/cpp/JSIContext.h
+++ b/node_modules/expo-modules-core/android/src/main/cpp/JSIContext.h
@@ -14,7 +14,11 @@
 #include <fbjni/fbjni.h>
 #include <jsi/jsi.h>
 #include <ReactCommon/CallInvokerHolder.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 #if IS_NEW_ARCHITECTURE_ENABLED
 
diff --git a/node_modules/expo-modules-core/android/src/main/cpp/JavaCallback.h b/node_modules/expo-modules-core/android/src/main/cpp/JavaCallback.h
index 0835676..71a1595 100644
--- a/node_modules/expo-modules-core/android/src/main/cpp/JavaCallback.h
+++ b/node_modules/expo-modules-core/android/src/main/cpp/JavaCallback.h
@@ -13,7 +13,11 @@
 #include <react/jni/WritableNativeArray.h>
 #include <react/jni/WritableNativeMap.h>
 #include <fbjni/detail/CoreClasses.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 #include <react/bridging/LongLivedObject.h>
 
 namespace jni = facebook::jni;
diff --git a/node_modules/expo-modules-core/android/src/main/cpp/JavaScriptRuntime.h b/node_modules/expo-modules-core/android/src/main/cpp/JavaScriptRuntime.h
index eb6f6d4..02c7b74 100644
--- a/node_modules/expo-modules-core/android/src/main/cpp/JavaScriptRuntime.h
+++ b/node_modules/expo-modules-core/android/src/main/cpp/JavaScriptRuntime.h
@@ -6,7 +6,11 @@
 
 #include <jsi/jsi.h>
 #include <fbjni/fbjni.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace jsi = facebook::jsi;
 namespace jni = facebook::jni;
diff --git a/node_modules/expo-modules-core/android/src/main/cpp/RuntimeHolder.h b/node_modules/expo-modules-core/android/src/main/cpp/RuntimeHolder.h
index c2d02a0..6912721 100644
--- a/node_modules/expo-modules-core/android/src/main/cpp/RuntimeHolder.h
+++ b/node_modules/expo-modules-core/android/src/main/cpp/RuntimeHolder.h
@@ -6,7 +6,11 @@
 #include <jsi/jsi.h>
 
 #include <ReactCommon/CallInvokerHolder.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace jni = facebook::jni;
 namespace jsi = facebook::jsi;
diff --git a/node_modules/expo-modules-core/common/cpp/BridgelessJSCallInvoker.h b/node_modules/expo-modules-core/common/cpp/BridgelessJSCallInvoker.h
index abacef3..fecfdfc 100644
--- a/node_modules/expo-modules-core/common/cpp/BridgelessJSCallInvoker.h
+++ b/node_modules/expo-modules-core/common/cpp/BridgelessJSCallInvoker.h
@@ -4,7 +4,11 @@
 
 #ifdef __cplusplus
 
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 #include <ReactCommon/RuntimeExecutor.h>
 
 namespace jsi = facebook::jsi;
diff --git a/node_modules/expo-modules-core/common/cpp/TestingSyncJSCallInvoker.h b/node_modules/expo-modules-core/common/cpp/TestingSyncJSCallInvoker.h
index 90d4035..2edf7a8 100644
--- a/node_modules/expo-modules-core/common/cpp/TestingSyncJSCallInvoker.h
+++ b/node_modules/expo-modules-core/common/cpp/TestingSyncJSCallInvoker.h
@@ -4,7 +4,11 @@
 
 #ifdef __cplusplus
 
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 
 #include <jsi/jsi.h>
 
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
index 6d9bcaa..268927d 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.h
@@ -7,7 +7,11 @@
 #import <jsi/jsi.h>
 
 #import <React/RCTBridgeModule.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 using namespace facebook;
 using namespace react;
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
index e924da4..db20a64 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIConversions.mm
@@ -10,6 +10,11 @@
 #import <ExpoModulesCore/EXJavaScriptSharedObjectBinding.h>
 #import <ExpoModulesCore/EXStringUtils.h>
 #import <Foundation/NSURL.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace expo {
 
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h b/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
index 407f57f..e8a5cf7 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.h
@@ -8,6 +8,15 @@
 #import <React/RCTBridgeModule.h>
 #import <ReactCommon/TurboModuleUtils.h>
 #import <ExpoModulesCore/ObjectDeallocator.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace jsi = facebook::jsi;
 namespace react = facebook::react;
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.mm b/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.mm
index a566e53..40fa841 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.mm
+++ b/node_modules/expo-modules-core/ios/JSI/EXJSIUtils.mm
@@ -7,6 +7,11 @@
 #import <ExpoModulesCore/JSIUtils.h>
 #import <ExpoModulesCore/NativeModule.h>
 #import <ExpoModulesCore/EventEmitter.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace expo {
 
diff --git a/node_modules/expo-modules-core/ios/JSI/EXJavaScriptRuntime.h b/node_modules/expo-modules-core/ios/JSI/EXJavaScriptRuntime.h
index a38de45..e1255a5 100644
--- a/node_modules/expo-modules-core/ios/JSI/EXJavaScriptRuntime.h
+++ b/node_modules/expo-modules-core/ios/JSI/EXJavaScriptRuntime.h
@@ -8,6 +8,15 @@
 #import <ExpoModulesCore/EXJavaScriptObject.h>
 
 #ifdef __cplusplus
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
 
 namespace jsi = facebook::jsi;
 namespace react = facebook::react;
diff --git a/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h b/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
index d7a5fdb..6f5f292 100644
--- a/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
+++ b/node_modules/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h
@@ -4,7 +4,11 @@
 
 #ifdef __cplusplus
 
+#if __has_include(<ReactCommon/CallInvoker.h>)
 #include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
 #include <ExpoModulesCore/MainThreadInvoker.h>
 
 #include <jsi/jsi.h>
