diff --git a/node_modules/react-native-worklets-core/ios/Worklets.mm b/node_modules/react-native-worklets-core/ios/Worklets.mm
index 403d88bf7d89f57ac18fdffe4b7aa839a62234b0..2e7bfa573210a55d01b53c121611678a700f4667 100644
--- a/node_modules/react-native-worklets-core/ios/Worklets.mm
+++ b/node_modules/react-native-worklets-core/ios/Worklets.mm
@@ -4,6 +4,12 @@
 #import <React/RCTBridge.h>
 #import <React/RCTUtils.h>
 #import <ReactCommon/RCTTurboModule.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#import <ReactCommon/CallInvoker.h>
+#else
+#import <React_callinvoker/CallInvoker.h>
+#endif
+
 
 @implementation Worklets
 
diff --git a/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.cpp b/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.cpp
index 9503d586b5893a16bc6d3c93d4458ad1b9f8094f..3af69c5a38aa34dd377c987489efb0d9748b349b 100644
--- a/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.cpp
+++ b/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.cpp
@@ -26,6 +26,12 @@
 
 #ifdef ANDROID
 #include <fbjni/fbjni.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
+
 #endif
 
 namespace RNWorklet {
diff --git a/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.h b/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.h
index 789e786bf6a17d7c793a7d852cceed5dd19d2b6a..31baeda52c2073fef67a00f868252a5453eacf7e 100644
--- a/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.h
+++ b/node_modules/react-native-worklets-core/cpp/WKTJsiWorkletContext.h
@@ -14,6 +14,12 @@
 #include <vector>
 
 #include <jsi/jsi.h>
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
+
 
 namespace RNWorklet {
 
diff --git a/node_modules/react-native-worklets-core/android/cpp-adapter.cpp b/node_modules/react-native-worklets-core/android/cpp-adapter.cpp
index ea6a539cb67cb89cd555548493bb08cadb32c3c1..08b64a91578a9eb7e6df8deb4236ee862482ef0f 100644
--- a/node_modules/react-native-worklets-core/android/cpp-adapter.cpp
+++ b/node_modules/react-native-worklets-core/android/cpp-adapter.cpp
@@ -4,6 +4,12 @@
 #include <ReactCommon/CallInvokerHolder.h>
 
 #include "WKTJsiWorkletApi.h"
+#if __has_include(<ReactCommon/CallInvoker.h>)
+#include <ReactCommon/CallInvoker.h>
+#else
+#include <React_callinvoker/CallInvoker.h>
+#endif
+
 
 using namespace facebook; // NOLINT
 
